skimage/transform/_thin_plate_splines.py

```diff
@@ -0,0 +1,294 @@
+import numpy as np
+import scipy as sp
+
+
+class TpsTransform:
+    def __init__(self):
+        self._estimated = False
+        self.parameters = None
+        self.control_points = None
```

Comment:
By the way, how did you come up with the term control points? I am currently trying to figure out the corresponding symbol in the paper.